website/ref/language.md

```diff
@@ -1675,21 +1675,21 @@ The following namespaces have special meanings to the language:
 
 ## Pre-Defined Modules
 
-Namespaces that are not special (i,e. one of the above) are also called
+Namespaces that are not special (i.e. one of the above) are also called
 **modules**. Aside from these special namespaces, Elvish also comes with the
-following modules:
+following modules that are always available, but might need to be imported by a
+`use` command:
```

Comment:
"always available" is duplicate with the first bullet and contradicts the second bullet. Better to say:

> Elvish also comes with the following modules that can be imported with `use`:

And make the `edit` bullet not a bullet, just a separate paragraph.

Reply:
Done.